### ISSUE TITLE
Intl.Locale: follow up tasks

### ISSUE THREAD

**Issue by rwaldron** [tc39/test262#1538]
These tasks need to be completed as a follow up to #1536

- [ ] Add `Intl.Locale` feature to features.txt
  - [ ] Update frontmatter for all of test/intl402/Locale/*
- [ ] Break up test/intl402/Locale/constructor-options.js by section
- [ ] Break up test/intl402/Locale/constructor-properties.js by aspect: 
  - [ ] test/intl402/Locale/prop-desc.js
  - [ ] test/intl402/Locale/name.js
  - [ ] test/intl402/Locale/length.js
  - [ ] test/intl402/Locale/prototype/prop-desc.js
  - [ ] test/intl402/Locale/prototype/name.js
  - [ ] test/intl402/Locale/prototype/length.js    
- [ ] Break up test/intl402/Locale/constructor-tag-type.js, one test per file (eg. [test/built-ins/Proxy/create-handler-not-object-throw-boolean.js](https://github.com/tc39/test262/blob/master/test/built-ins/Proxy/create-handler-not-object-throw-boolean.js) and others)
  - [ ] invalid-tag-throws-boolean.js
  - [ ] invalid-tag-throws-function.js
  - [ ] invalid-tag-throws-null.js
  - [ ] invalid-tag-throws-number.js
  - [ ] invalid-tag-throws-symbol.js
  - [ ] invalid-tag-throws-undefined.js

**Comment by littledan** [comment]
Cc @ms2ger

**Comment by littledan** [comment]
There is also missing coverage for the various getters, toString, and using a Locale object in another formatter constructor (even if the toString method has been messed with).

**Comment by rwaldron** [comment]
@littledan The point of making this a separate follow up issue was to avoid imposing further burden on a new contributor.

**Comment by littledan** [comment]
@rwaldron I agree with this goal. I'm documenting missing coverage to help the contributor. Do you have a suggestion for a friendlier manner/place to do this?

**Comment by leobalter** [comment]
Some suggestions to follow up with:

- [ ] use the verifyProperty helper to assert the given descriptor matches the expected behavior as well
- [ ] Use `esid: sec-intl.locale` to replace the `pending` parts
- [ ] Prefer the ordinary function format to avoid cross features compatibility issues. e.g.: https://github.com/tc39/test262/pull/1536/files#diff-aacbf8c0e57d2ca04db39550f8d882e8R14
- [ ] Use static assert calls, avoid DRY for tests. e.g.: https://github.com/tc39/test262/pull/1536/files#diff-bb5ab846209e801373de2cecfc6e362bR74

**Comment by leobalter** [comment]
It also seems to be missing:

- [ ] tests for the get access properties from the Intl.Locale.prototype. e.g. https://tc39.github.io/proposal-intl-locale/#sec-Intl.Locale.prototype.calendar
